src/Menu/Menu.js

```diff
@@ -1,7 +1,7 @@
 // @flow weak
 
 import React, { Component, PropTypes } from 'react';
-import { findDOMNode } from 'react-dom';
+import reactDomUtil from 'react-dom';
```

Comment:
The community convention is to use `ReactDOM `. Can we stick to that? For instance https://github.com/facebook/react#examples